base/timing.jl

```diff
@@ -430,6 +430,33 @@ macro allocated(ex)
     end
 end
 
+"""
+    @allocs
+
+A macro to evaluate an expression, discard the resulting value, and instead return the
+total number of allocations during evaluation of the expression.
+
+See also [`@allocated`](@ref), [`@time`](@ref), [`@timev`](@ref), [`@timed`](@ref),
+and [`@elapsed`](@ref).
+
+```julia-repl
+julia> @allocs rand(10^6)
```

Comment:
i just copied the example from `@allocated, happy to change or add others if people prefer... it's not obvious to me why the answer here is `2` not `1` 😂 which may make it a good or a bad example depending on your point of view